taskcluster/scripts/build/windows.ps1

```diff
@@ -34,12 +40,24 @@ Copy-Item -Path $env:VCToolsRedistDir\\MergeModules\\Microsoft_VC143_CRT_x86.msm
 python3 ./scripts/utils/generate_glean.py
 python3 ./scripts/utils/import_languages.py
 
+#Do not continune from this point on when we encounter an error
+$ErrorActionPreference = "Stop"
+
+# Actually compile!
 ./scripts/windows/compile.bat --nmake
+# Copies all relevant files into unsigned/
 nmake install 
 
-Write-Output "Writing Artifacts"
-
+# This should not be a thing, yet that's happening..
```

Comment:
The checkout repo is often cached across tasks to speed up performance, so I'd guess that's what's happening here.

I actually fixed this on my WIP windows signing branch by moving the artifacts dir to `TASK_WORKDIR` rather than in the repo:
https://github.com/mozilla-releng/staging-mozilla-vpn-client/blob/releases/ahal/win_signing/taskcluster/scripts/build/windows.ps1#L43

Feel free to steal my changes from there if you like.